pygmt/__init__.py

```diff
@@ -1,11 +1,17 @@
 # pylint: disable=missing-docstring
 #
-# The main API for PyGMT.
+# PyGMT is a library for processing geospatial and geophysical data and making
```

Comment:
How about changing the order of the first and second sentences? because I think what PyGMT can do actually depends on GMT so it is better to point that PyGMT is GMT's Python interface first.

Reply:
Would be fine for me, so far I used the description we currently have on https://github.com/GenericMappingTools/pygmt. Any further opinions?

Reply:
I am thinking about GMT's main features/functionalities. The description, `processing geospatial and geophysical data`, is fine, but people who first find GMT may just regard GMT as a geophysical tool.

Have a look at the landing page of GMT (https://www.generic-mapping-tools.org/):
> The Generic Mapping Tools (GMT) are widely used across the Earth, Ocean, and Planetary sciences and beyond. A diverse community uses GMT to process data, generate publication-quality illustrations, automate workflows, and make animations.

> Whether Cartesian, geographic, or time-series, GMT can process your data.

I can see the following key points:
- GMT can be used to process data. What're kinds of data? Cartesian, geographic, or time-series data.
- GMT can be used to generate publication-quality illustrations and make animations.
- GMT supports automatic workflow. Can we call it "scripting plotting"?

So, do we want to include some of those points in both the **package docstring** and the [about section in README.rst](https://github.com/GenericMappingTools/pygmt#about)?

By the way, shall we replace "Earth sciences" by "Earth, Ocean, and Planetary sciences"?
> It provides a Pythonic interface for the Generic Mapping Tools (GMT), a command-line program widely used in the Earth Sciences.

Reply:
Agree with you @core-man. I made some adjustments.

Reply:
> 
> 
> 
> By the way, shall we replace "Earth sciences" by "Earth, Ocean, and Planetary sciences" which is used at the GMT landing page?
> 
> > It provides a Pythonic interface for the Generic Mapping Tools (GMT), a command-line program widely used in the Earth Sciences.

If we want to change this for PyGMT, I would recommend to change it for all GMT related content.

Reply:
> > By the way, shall we replace "Earth sciences" by "Earth, Ocean, and Planetary sciences" which is used at the GMT landing page?
> > > It provides a Pythonic interface for the Generic Mapping Tools (GMT), a command-line program widely used in the Earth Sciences.
> 
> If we want to change this for PyGMT, I would recommend to change it for all GMT related content.

I think that this change throughout all GMT related content would be welcome, but could be handled separately from this PR.

Reply:
> 
> 
> > > By the way, shall we replace "Earth sciences" by "Earth, Ocean, and Planetary sciences" which is used at the GMT landing page?
> > > > It provides a Pythonic interface for the Generic Mapping Tools (GMT), a command-line program widely used in the Earth Sciences.
> > 
> > 
> > If we want to change this for PyGMT, I would recommend to change it for all GMT related content.
> 
> I think that this change throughout all GMT related content would be welcome, but could be handled separately from this PR.

Of course 😉. Only wanted to mention it at this point.